src/views/DatasetManager/components/CreateDatasetButton/components/DatasetCreationParameters/DatasetCreationParameters.js

```diff
@@ -107,7 +101,7 @@ export const DatasetCreationParameters = ({ dataSourceRunTemplates, parentDatase
                   gridItemProps={{ xs: 12, sx: { pt: 2 } }}
                   parameterData={parameter}
                   context={{ editMode: true, targetDatasetId: parentDataset?.id }}
-                  parameterValue={value}
+                  parameterValue={Array.isArray(value) ? value : value?.split(',')}
```

Comment:
I wonder why we have to parse the string value here, whereas the other types of parameters don't need any conversion. We even have a [conversion mechanism for the list type](https://github.com/Cosmo-Tech/azure-sample-webapp/blob/experimental/on_premises_api_v4/src/utils/scenarioParameters/generic/ConversionFromString.js#L48), so it should always be stored as an Array in memory, right?

Reply:
I honestly don't remember why I did it :thinking: